cmd/snap/cmd_sign.go

```diff
@@ -100,9 +107,87 @@ func (x *cmdSign) Execute(args []string) error {
 		return err
 	}
 
-	_, err = Stdout.Write(encodedAssert)
+	outBuf := bytes.NewBuffer(nil)
+
+	_, err = outBuf.Write(encodedAssert)
+	if err != nil {
+		return err
+	}
```

Comment:
I think this can use an asserts.Encoder now:

enc := asserts.NewEncoder(outBuf)
... enc.WriteEncoded(encodedAssert)
... enc.Encode(otherAssert)

this avoid needing to add separators by hand